Uchu.World/Objects/Components/Player/ModularBuilderComponent.cs

```diff
@@ -132,15 +132,17 @@ public async Task ConfirmFinish()
                 return;
 
             var inventory = GameObject.GetComponent<InventoryManagerComponent>();
-            
-            foreach (var temp in inventory[InventoryType.TemporaryModels].Items)
+            if (inventory[InventoryType.TemporaryModels] != null)
             {
-                await inventory.MoveItemBetweenInventoriesAsync(
-                    temp,
-                    temp.Count,
-                    InventoryType.TemporaryModels,
-                    InventoryType.Models
-                );
+            foreach (var temp in inventory[InventoryType.TemporaryModels].Items)
```

Comment:
The `foreach` should be tabbed 1 more time (4 more spaces). It is in line with the `if` statement than the curly braces for the loop.

Reply:
Ahh, my bad. Thanks for letting me know.

Reply:
If you looked at the new commit on GitHub, you may have noticed that the spacing is *way* off. I am guessing the text editor/IDE you used didn't use 4 spaces when pressing tab and replaced the line with tabs.

Reply:
Yikes. Yeah, it certainly isn't showing up like that in my IDE. My apologies, let me go ahead and fix this in GitHub web.

Reply:
I hope I got it right this time....

Reply:
It looks right in the edit view but not in the commit view because of how tabs are rendered. The line still uses 2 tabs instead of 12 spaces, which I am guessing is because the IDE *replaced* the spaces with tabs.

Reply:
I see. Should I revert these commits and try again?

Reply:
I made the change using the GitHub editor.